src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -614,6 +614,12 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Corax.VectorSearch.DefaultNumberOfCandidatesForQuerying", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public int CoraxVectorDefaultNumberOfCandidatesForQuerying { get; protected set; }
         
+        [Description("Order by score automatically when vector.search is inside query.")]
+        [DefaultValue(true)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.Corax.OrderByScoreAutomaticallyWhenVectorSearchIsUsed", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
```

Comment:
Shouldn't this be inside QueryConfiguration?

Reply:
QueryConfiguration is global; we would like to have it per index as well.

Reply:
Yes, we already have other `Indexing.OrderBy...` settings defined here